src/Mapping/Annotation/Slug.php

```diff
@@ -44,14 +44,20 @@ final class Slug implements GedmoAnnotation
     public string $separator = '-';
     public string $prefix = '';
     public string $suffix = '';
-    /** @var SlugHandler[] */
+
+    /**
+     * @var SlugHandler[]
+     *
+     * @deprecated since since gedmo/doctrine-extensions 3.17
+     */
     public $handlers = [];
+
     public string $dateFormat = 'Y-m-d-H:i';
 
     /**
      * @param array<string, mixed> $data
      * @param string[]             $fields
-     * @param SlugHandler[]        $handlers
+     * @param SlugHandler[]        $handlers @deprecated since since gedmo/doctrine-extensions 3.17
```

Comment:
I didn't use `@deprecated` annotation on top of others annotations. Can you confirm it generates the expected `E_USER_DEPRECATED` error?

Reply:
The `@deprecated` annotation on its own won't trigger a runtime deprecation here, and the attribute driver already ignores this property entirely.  So I think we've got as much as we can get out of this one without adding a check that ends up throwing an exception in the attribute driver if this is a non-empty array.

Reply:
Thanks for the explanation.
If this annotation is not used to trigger an `E_USER_DEPRECATED` error, I assume the only purpose here is documentary. If this is the case, I think we should keep the docblock structure as standardized as possible (avoiding more than 1 annotation per line).
Perhaps it is enough to add a description that explains the status of this argument.

Reply:
I just changed it to not use the annotation.  I've seen some projects handle param deprecations that way, but it's still not really a solved problem in PHP (the closest thing I could find in phpDocumentor is https://github.com/phpDocumentor/phpDocumentor/issues/1604 and how to deal with virtual method/property deprecations, but neither it nor any of the WIP PSR's for doc blocks get into this scenario).